package proto_ff;

import "yd_fieldoptions.proto";


message E_TaskTask
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_minlv = 2[(yd_fieldoptions.field_cname) = "发放等级"];
	optional int32 m_truelv = 3[(yd_fieldoptions.field_cname) = "可完成等级"];
	optional int32 m_type = 4[(yd_fieldoptions.field_cname) = "任务类型"];
	optional string m_showtype = 5[(yd_fieldoptions.field_cname) = "显示类型", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_weights = 6[(yd_fieldoptions.field_cname) = "排序权重"];
	optional string m_name = 7[(yd_fieldoptions.field_cname) = "任务名", (yd_fieldoptions.field_bufsize) = 64];
	optional string m_rececond = 8[(yd_fieldoptions.field_cname) = "接取条件", (yd_fieldoptions.field_bufsize) = 64];
	optional string m_receadd = 9[(yd_fieldoptions.field_cname) = "过程", (yd_fieldoptions.field_bufsize) = 64];
	optional string m_comcond = 10[(yd_fieldoptions.field_cname) = "完成条件", (yd_fieldoptions.field_bufsize) = 256];
	optional int32 m_nexttask = 11[(yd_fieldoptions.field_cname) = "后续任务"];
	optional string m_tracker = 12[(yd_fieldoptions.field_cname) = "任务追踪", (yd_fieldoptions.field_bufsize) = 64];
	optional string m_subtype = 13[(yd_fieldoptions.field_cname) = "提交方式", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_subnpc = 14[(yd_fieldoptions.field_cname) = "npc的ID"];
	optional int32 m_subaward = 15[(yd_fieldoptions.field_cname) = "提交奖励"];
}


message Sheet_TaskTask
{
	repeated E_TaskTask E_TaskTask_List = 1[(yd_fieldoptions.field_arysize)=512];
}
